packages/resource-deployment/scripts/install.sh

```diff
@@ -80,11 +80,12 @@ while getopts ":r:s:l:e:o:p:c:t:" option; do
     p) publisherEmail=${OPTARG} ;;
     c) webApiAdClientId=${OPTARG} ;;
     t) webApiAdClientSecret=${OPTARG} ;;
+    v) releaseVersion=${OPTARG} ;;
     *) exitWithUsageInfo ;;
     esac
 done
 
-if [[ -z $resourceGroupName ]] || [[ -z $subscription ]] || [[ -z $environment ]] || [[ -z $orgName ]] || [[ -z $publisherEmail ]] || [[ -z $webApiAdClientId ]] || [[ -z $webApiAdClientSecret ]]; then
+if [[ -z $resourceGroupName ]] || [[ -z $subscription ]] || [[ -z $environment ]] || [[ -z $orgName ]] || [[ -z $publisherEmail ]] || [[ -z $webApiAdClientId ]] || [[ -z $webApiAdClientSecret ]] || [[ -z $releaseVersion ]]; then
```

Comment:
We are going to pass the predefined release id right?

Reply:
yes, the release id from the pipeline